homeassistant/components/plugwise/config_flow.py

```diff
@@ -159,11 +145,22 @@ async def async_step_zeroconf(
                 "configuration_url": (
                     f"http://{discovery_info.host}:{discovery_info.port}"
                 ),
-                "product": _product,
             }
         )
         return await self.async_step_user()
 
+    def is_matching(self, other_flow: Self) -> bool:
```

Comment:
@emontnemery It looks like this function is never called?

Reply:
Never mind, I did some digging, I now understand how this is supposed to work :)